activerecord/lib/active_record/connection_adapters/mysql/schema_statements.rb

```diff
@@ -154,6 +154,10 @@ def default_row_format
             @default_row_format
           end
 
+          def valid_primary_key_option_keys
```

Comment:
```suggestion
          def valid_primary_key_options
```

I think we could make the whole thing a litle bit less verbose by using a shorter naming conventions.

e.g.

  - `def valid_<thing>_options`
  - `VALID_<THING>_options`

But also drop the constants when they don't help DRYing up.